### PR TITLE
vagrant-manager: Add version 2.3.2

### PR DESCRIPTION
**Pull request by absalomedia**
Add GUI based Vagrant Manager to Scoop

### CODE REVIEW

File: bucket/vagrant-manager.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/absalomedia/vagrant-manager",
+        "regex": "/tag/([-\\w.]+)"
+    },
+    "autoupdate": {
+        "url": "https://github.com/absalomedia/vagrant-manager/releases/download/$version/vagrant-manager.exe"
+    }
+}
```
Comment (Ash258) [inline bucket/vagrant-manager.json]:
```suggestion
}

```

File: bucket/vagrant-manager.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "shortcuts": [
+        [
+            "vagrant-manager.exe",
+            "Vagrant Manager"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/absalomedia/vagrant-manager",
+        "regex": "/tag/([-\\w.]+)"
+    },
```
Comment (Ash258) [inline bucket/vagrant-manager.json]:
```suggestion
    "checkver": "github",
```

File: bucket/vagrant-manager.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+ {
+    "version": "2.3.2",
+    "description": "Vagrant Manager",
```
Comment (Ash258) [inline bucket/vagrant-manager.json]:
```suggestion
    "description": "Status bar menu for managing all Vagrant machines from one central location",
```

**Comment by Ash258** [comment]
Why did you select exactly this fork?

File: bucket/vagrant-manager.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/absalomedia/vagrant-manager",
+        "regex": "/tag/([-\\w.]+)"
+    },
+    "autoupdate": {
+        "url": "https://github.com/absalomedia/vagrant-manager/releases/download/$version/vagrant-manager.exe"
+    }
+}
```
Comment (Ash258) [inline bucket/vagrant-manager.json]:
```suggestion
    "suggest": {
        "Vagrant": "vagrant"
    },
    "url": "https://github.com/absalomedia/vagrant-manager/releases/download/2.3.2/vagrant-manager.exe#/dl.7z",
    "hash": "c6a99b2e5b18f384ac5b25abc539c319efe9b3cd472c9b73a01130826b1f6240",
    "architecture": {
        "64bit": {
            "installer": {
                "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
            }
        },
        "32bit": {
            "installer": {
                "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\""
            }
        }
    },
    "post_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse",
    "shortcuts": [
        [
            "Vagrant Manager.exe",
            "Vagrant Manager"
        ]
    ],
    "checkver": "github",
    "autoupdate": {
        "url": "https://github.com/absalomedia/vagrant-manager/releases/download/$version/vagrant-manager.exe#/dl.7z"
    }
```

**Comment by absalomedia** [comment]
> Why did you select exactly this fork?

I'm the maintainer for the fork. The project I forked off was not maintained so in that time I've:

- rewritten underlying foundations & logic
- implemented cross platform build processes (Windows, OSX, Linux), including automated version releases. 7z files aren't meant to be captured as part of Travis yet Github is somehow doing that. EXE, DMG, ZIP, DEB files are the expected artifacts with the Travis build sequence.
- translated the app into multiple languages
- extended out the base functionality to include a much larger set of Vagrant related functions

File: bucket/vagrant-manager.json (hunk 4 above)
Comment (absalomedia) [reply bucket/vagrant-manager.json]: I downloaded the vagrant-manager-2.3.2-x64.nsis.7z package to verify it. Extracted out fine. Ran fine. Windows 10 Pro running v1809. 

The Vagrant Manager.exe file inside the NSIS 7z file did what it was supposed to do. End up in the Windows taskbar. It's not an installer, it's the final app.

Comment (Ash258) [reply bucket/vagrant-manager.json]: Yes I know it is not installer... And yes it is executed in taskbar, but right click do not work. It do nothing

- Sandbox 1903
- Server 2019
- Server 2012R2
- Windows 20H1
- Windows 1909

Comment (absalomedia) [reply bucket/vagrant-manager.json]: I'll check on 1909 once the feature update goes through on my machine tonight.

Comment (Ash258) [reply bucket/vagrant-manager.json]: Apply this changes, since current manifest is not working.

Comment (absalomedia) [reply bucket/vagrant-manager.json]: Applied changed to JSON based on your feedback

**Comment by absalomedia** [comment]
@Ash258 just for future reference, what is the hash based on? The 64-bit 7z or the 32bit?

**Comment by r15ch13** [comment]
Why not use `vagrant-manager-x64.7z` and `vagrant-manager-ia32.7z` instead of the whole installer (which is double the size)?

**Comment by Ash258** [comment]
https://github.com/lukesampson/scoop-extras/pull/3567#discussion_r375411907 I cannot make it work in my environemnt

**Comment by r15ch13** [comment]
okay 😄

**Comment by Ash258** [comment]
I will look into it in future, but right now it is not needed.

**Pull request merged**